vector/src/main/java/im/vector/app/features/login/LoginViewModel.kt

```diff
@@ -26,9 +26,6 @@ import com.airbnb.mvrx.Uninitialized
 import dagger.assisted.Assisted
 import dagger.assisted.AssistedFactory
 import dagger.assisted.AssistedInject
-import fr.gouv.tchap.features.platform.GetPlatformResult
```

Comment:
this file is not used anymore in Tchap, I reapplied the code coming from Element